src/components/v1/account/dao/index.js

```diff
@@ -1,15 +1,19 @@
-import accountModel from '../../../../models/account';
+import accountModel from "../../../../models/account";
 
-const findAccountByName = ({ name }) => accountModel
-  .findOne({
-    name,
-  })
-  .lean()
-  .exec();
+const findAccountByName = ({ name }) =>
```

Comment:
if just is one parameter is not necessary pass this as an object @raiandexter0607

Reply:
Thank you. This was solved.